doc/python_api_reference_vDev.md

```diff
@@ -7116,6 +7167,43 @@ def to_circuit(
     """
 ```
 
+<a name="stim.Tableau.to_pauli_string"></a>
+```python
+# stim.Tableau.to_pauli_string
+
+# (in class stim.Tableau)
+def to_pauli_string(
+    self,
+) -> stim.PauliString:
+    """Return a Pauli string equivalent to the tableau.
+
+    If the tableau fixes each pauli, creates an equivalent
```

Comment:
"fixes each Pauli" isn't defined. I think just say if it's equivalent to a Pauli product

Reply:
:+1: